### PR TITLE
Linkedin login

### PR DESCRIPTION
**Pull request by anithaamarnath**

### CODE REVIEW

File: src/auth/linkedinPassport.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  
+    clientID: process.env.linkedinClientID  as string,
+    clientSecret: process.env.linkedinClientSecret as string,
+    callbackURL: '/auth/linkedin/redirect',
+    scope: [ 'r_emailaddress','r_liteprofile'] 
+    
+  }
+  
+  const linkedinCallback = async (accessToken, refreshToken, profile, done) => {
+      console.log("hi this is profile",profile);
```
Comment (harlyon) [inline src/auth/linkedinPassport.ts:19]: We might not need the console logs anymore

Comment (anithaamarnath) [reply src/auth/linkedinPassport.ts]: review my PR

**Review by harlyon — APPROVED** [review]

**Review by justmic007 — APPROVED** [review]

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,7 @@
+jwtSecret=
```
Comment (justmic007) [inline .env.example:1]: Please remove the `**.env.example**` file

File: src/auth/linkedinPassport.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+          lastLogin: date,
+        };
+        const persistedUser = await UsersQueries.addUser(user);
+        done(null, persistedUser);
+      }
+    } catch (error) {
+      throw new Error("Internal Server Error");
+    }
+  }
+
```
Comment (justmic007) [inline src/auth/linkedinPassport.ts:45]: close the multiple spaces

**Pull request merged**